scripts/README.md

```diff
@@ -0,0 +1,24 @@
+# Scripts
+
+See [https://github.com/github/scripts-to-rule-them-all](https://github.com/github/scripts-to-rule-them-all)
+for additional background on these scripts.
+
+Below is a list of scripts available, along with a simple description of
+what each one does. The details of what they are doing is available within the
+script.
+
+[`pre-bootstrap`](pre-bootstrap)
+Directs towards base development machine setup.
+
+[`bootstrap`](bootstrap)
+Installs project's direct dependencies e.g. npm packages.
+
+[`start`](start)
+Starts the application a Docker container. Available at:
+`http://localhost:3000`
```

Comment:
The readme file says the default port is 3000, but there is no default set in the code. If PORT is not set you get the message 'app listening on port undefined'